client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -252,19 +259,25 @@ class Upload extends React.Component {
 			themeId,
 			upgradeJetpack,
 			backPath,
+			isBusiness,
+			isJetpack
 		} = this.props;
 
-		const showEligibility = ! this.props.isJetpackSite && this.state.showEligibility;
+		const showEligibility = ! isJetpack && this.state.showEligibility;
 
 		return (
 			<Main>
-				<QueryEligibility siteId={ siteId } />
 				<QueryActiveTheme siteId={ siteId } />
 				{ themeId && complete && <QueryTheme siteId={ siteId } themeId={ themeId } /> }
 				<ThanksModal
 					site={ selectedSite }
 					source="upload" />
 				<HeaderCake backHref={ backPath }>{ translate( 'Upload theme' ) }</HeaderCake>
+				{ ! isBusiness && ! isJetpack && <Banner
+					feature={ FEATURE_UPLOAD_THEMES }
+					plan={ PLAN_BUSINESS }
+					title={ translate( 'To upload themes, upgrade to Business Plan' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Upgrade to the Business plan to install plugins.' )` *ES Score: 5.67*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/upload-upgrade-nudge)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).